playground/backend/internal/cache/redis_cache.go

```diff
@@ -0,0 +1,148 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I believe we have a lot of things here to test. We can mock Redis and then check everything we do here.